### PR TITLE
Module-5: wrap up and week ahead

### PR DESCRIPTION
**Pull request by uditvira**
@darkdrgn2k I've suggested a few homework exercises. Please see if they make sense with your content :)

### CODE REVIEW

File: module-5.md [hunk 1]
```diff
@@ -125,4 +125,9 @@ Q & A about sharing broken Internet experiences, understanding failure points an
 
 ## 4: Wrap-up and week ahead
 
-TODO
+1. Watch the National Film Board series on Antennas:
+   * [Propogation](https://www.youtube.com/watch?v=7bDyA5t1ldU)
+   * [Directivity](https://www.youtube.com/watch?v=md7GjQQ2YA0&t=648s)
```
Comment (benhylau) [inline module-5.md]: `t=648s` intentional?

File: module-5.md [hunk 2]
```diff
@@ -125,4 +125,9 @@ Q & A about sharing broken Internet experiences, understanding failure points an
...
+   * [Bandwidth](https://www.youtube.com/watch?v=9iV_YICgifA&t=566s)
```
Comment (benhylau) [inline module-5.md]: `t= 566s ` intentional?

**Comment by benhylau** [comment]
Can the videos be optionals?

**Comment by benhylau** [comment]
@uditvira ^

**Comment by uditvira** [comment]
@benhylau yes, I made the videos optional and removed the unintentional start-times on the youtube links

**Pull request merged**